doc/nrf/libraries/others/sfloat.rst

```diff
@@ -0,0 +1,54 @@
+.. _lib_sfloat:
+
+Short float (SFLOAT)
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The library provides an implementation of the SFLOAT type.
```

Comment:
Library or module?

Reply:
I would say it is a library because it resides in the ```nrf/lib``` directory